# Cubic polynomial conditions: intial and final position, velocity, and time
float32 p0
float32 pf
float32 v0
float32 vf
float32 t0
float32 tf